src/core/_modules/global/modules/dns/outputs.tf

```diff
@@ -0,0 +1,23 @@
+output "public_dns_zones" {
+  value = {
+    io = {
+      name = length(azurerm_dns_zone.io_pagopa_it) > 0 ? azurerm_dns_zone.io_pagopa_it[0].name : null
```

Comment:
Do you think we can remove all the `count`s? I can't see how those conditions could be false

Reply:
Agree! In fact, I don't know why counts are also present on the resources. What do you think if we remove them also there?
I guess it's been done to allow deployment in other environments, but if it's not a practice anymore, I would remove them

Reply:
I agree, proceed (even in the case of multiple environments, they would be managed differently now)